code/game/objects/items/weapons/power_cells.dm

```diff
@@ -166,3 +157,22 @@
 
 	reset_vars_after_duration(resettable_vars, duration)
 
+
+/obj/item/weapon/cell/infinite
+	name = "infinite-capacity power cell!"
+	icon_state = "icell"
+	origin_tech = Tc_POWERSTORAGE + "=6;" + Tc_NANOTRASEN + "=5"
+	maxcharge = 35000
+	starting_materials = list(MAT_IRON = 700, MAT_GLASS = 80)
+
+/obj/item/weapon/cell/infinite/New()
+	..()
+	desc = "This cell is the latest in NT technology, having the capability to perpetually recharge itself. It has a power rating of Infinity!"
```

Comment:
Or just
Add a line that says that to their `examine()` and don't fuck with the `desc`